login-page.html

```diff
@@ -0,0 +1,68 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=1920px, initial-scale=1.0">
+        <link rel="stylesheet" href="css/login-page.css">
+        <title>LOGIN</title>
+        <link rel="icon" href="images/favicon.ico">
+    </head>
+    <body class="login-container">
+        <div class="login-content">
+
+            <img src="images/login-page/aksu.png" alt="err">
+            <img class="login-title" src="images/login-page/title.png" alt="err">
+
+            <h1 class="login-subtitle">LOGIN ACCOUNT</h1>
+
+            <div class="login-form">
+                <form action="name">
+
+                    <p class="err1"></p>
+                    <label class="label-login" for="fname">username</label>
+                    <input type="text" class="fname" name="username">
+                    
+                    <label class="label-login" for="password">password</label>
+                    <input type="password" class="password" name="raw-password">
+                    
+                </form>
+                <button class="button-page">LOGIN</button>
+            </div>
+
+            <div class="singIn-content">
+                <div class="singIn-container">
+
+                    <img class="img-singIn" src="images/login-page/back-jbt.jpeg" alt="sorry">
+                    <h1 class="title-singIn">Don't have an account?</h1>
+                    <p>Don't worry, just</p>
+                    <button class="button-page">SIGN UP</button>
+
+                </div>   
+            </div>
+        </div>
+
+        <script type="text/javascript">
+                
+            window.onload = function (){
+
+                document.querySelector(".button-page").onclick = ()=>{
```

Comment:
`button-page` is not a really good name... Use something like `login-page-button`
Notice that you have 2 items with that same class.
Right now it works because you are applying the event to the first one.
In case you need to use querySelector/querySelectorAll use unique class names.
- "login-button"
- "sign-up-button"

If you have shared style you could do something like that:
```css
.login-button, .sign-up-button{
// Shared attributes....
}
```
Otherwise you can create a class and assign that to the buttons other then the ones mentioned ahead:
```html
<button class="login-page-button login-button">LOGIN</button>
<button class="login-page-button sign-up-button">Sign Up</button>
```

I personally like more the last approach ^^^^^^^